test/integration/index.js

```diff
@@ -1,9 +1,10 @@
-/* global $, afterEach, beforeEach, browser, describe, it */
+/* global afterEach, beforeEach, describe, it */
 
 const { mkdir } = require('fs/promises')
 const { dirname, join } = require('path')
 const { cwd } = require('process')
-const { expect } = require('chai')
+const { $, browser, expect } = require('@wdio/globals')
```

Comment:
**suggestion** Looks like WebDriver provides its own `expect` function, however it has a different syntax for its matchers than `chai`. That means we now need to juggle two syntaxes depending on whether we're writing integration or functional tests. If we swap, I'd be keep to make the jump fully and take advantage of WebDriver's specialised matchers like, [`toBeDisplayed`](https://webdriver.io/docs/api/expect-webdriverio/#tobedisplayed), [`toBeFocused`](https://webdriver.io/docs/api/expect-webdriverio/#tobefocused),...

Reply:
Liking that we don't have `$` and `browser` as globals anymore and we know explicitely where they come from, though 😍

Reply:
Cheer @romaricpascal

Is it a blocker?